src/js/components/Video/__tests__/Video-test.js

```diff
@@ -67,12 +67,12 @@ describe('Video', () => {
   });
 
   test('controls below renders', () => {
-    const { container } = render(<App controls="below" />);
```

Comment:
Let's leave these tests alone, to ensure we haven't broken existing behavior, and then add new tests for the new style of property value.